File: models/book.py
from dataclasses import dataclass
from datetime import datetime

@dataclass
class Book:
    id: int | None = None
    title: str | None = None
    summary: str | None = None
    genre: str | None = None
    word_count: int | None = None
    author_username: str | None = None
    create_time: datetime | None = None
    update_time: datetime | None = None
    status: str | None = None
    cover: str | None = None

File: src/main/java/org/xu/novel/domain/po/Book.java
package org.xu.novel.domain.po;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Book {
    private Integer id;
    private String title;
    private String summary;
    private String genre;
    private Integer wordCount;
    private String authorUsername;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;
    private String status;
    private String cover;
}